core/domain/question_services.py

```diff
@@ -0,0 +1,74 @@
+# coding: utf-8
+#
+# Copyright 2017 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Services for questions data model."""
+
+from core.platform import models
+
+import feconf
+
+(question_models,) = models.Registry.import_models([models.NAMES.question])
+
+
+def _create_question(committer_id, question, commit_message):
+    """Creates a new question.
+
+    Args:
+        committer_id: str. ID of the committer.
+        question: Question. question domain object.
+        commit_message: str. A description of changes made to the question.
+    TODO: implement commit_cmds.
```

Comment:
Please, no -- implement them in this PR. It's core functionality that enables us to reconstruct the history of edits, and if we are already implementing functions like create/update then this PR is incomplete without it.

Reply:
Well, I have not written the functionality of update in this PR. I take from this comment that I have to implement update functionality as well. :P 
This will require me to implement next milestone in the `[doc](https://docs.google.com/document/d/1_EMD4JVn2H9aBX8OkkbieeXOcM6mx5EaB2tkHOiQ4tI/edit?ts=586f16a9#heading=h.7rd4a08c9xp4)`. This PR will cover 4/6 milestones (old) or 3/5 milestones as per new.

Reply:
Oh, no, sorry -- you don't need to implement update functionality in this PR. It's fine to add that as a new, separate thing once we've got the base design set up.

Reply:
I have implemented update functionality in this PR now and i think that will better implement the QuestionChangeList methods.

Reply:
The original comment in this thread has not been addressed. The TODO here should be resolved prior to submission.